tests/test_geobox.py

```diff
@@ -145,36 +147,37 @@ def test_geobox():
 
         assert GeoBox.from_geopolygon(polygon, resolution, crs=geobox.crs) == geobox
 
-        assert abs(resolution[0]) > abs(
+        # check that extra padding added by alignment is smaller than pixel size
+        assert abs(resolution.x) > abs(
```

Comment:
note that this changes the test, it was incorrectly using Y resolution to check for padding change in X axis, but since both resolutions are of the same magnitude it didn't cause problems. So it's not a very good test, need a test with non-square pixels to check padding correctness.